src/cursor/cur_table.c

```diff
@@ -804,6 +811,30 @@ __wt_table_range_truncate(WT_CURSOR_TABLE *start, WT_CURSOR_TABLE *stop)
     return (ret);
 }
 
+/*
+ * __wt_cursor_bounds_save --
+ *     Save cursor bounds to restore the state.
+ */
+static inline void
+__wt_cursor_bounds_save(WT_CURSOR *cursor, WT_CURSOR_BOUNDS_STATE *state)
```

Comment:
Currently these aren't called from outside of this file so the `__wt` isn't required. I think we may need to for the curindex work however in which case I would suggest you move it to cur_std.c and keep the `__wt`.

Reply:
Move to cur_std.c

Reply:
[59a73d6](https://github.com/wiredtiger/wiredtiger/pull/8265/commits/59a73d6177d493f37cd1d9b9e8a5f5b0b9edb277)